cob_moveit_bringup/launch/moveit_ompl.rviz

```diff
@@ -5,9 +5,12 @@ Panels:
     Property Tree Widget:
```

Comment:
do not commit changes to the rviz config files...keep your configuration local and uncommited
same below